CTOR-990 : Refresh network-switchs-hirschmann-snmp (#3696)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/network-switchs-hirschmann-snmp.md

---

> id: network-switchs-hirschmann

Author: ```suggestion
id: network-switchs-hirschmann-snmp
```

The following monitoring connectors will be installed when you install the **Hirschmann switch** connector through the
**Configuration > Monitoring Connector Manager** menu:
* [Base Pack](./base-generic.md)

## Pack assets

### Templates

The Monitoring Connector **Hirschmann switch** brings a host template:

* **Net-Hirschmann-Standard-SNMP-custom**

The connector brings the following service templates (sorted by the host template they are attached to):

<Tabs groupId="sync">
<TabItem value="Net-Hirschmann-Standard-SNMP-custom" label="Net-Hirschmann-Standard-SNMP-custom">

| Service Alias | Service Template                              | Service Description                                 |
|:--------------|:----------------------------------------------|:----------------------------------------------------|
| Cpu           | Net-Hirschmann-Standard-Cpu-SNMP-custom       | Check the rate of utilization of CPU for the device |
| Hardware      | Net-Hirschmann-Standard-Hardware-SNMP-custom  | Check hardware environment                          |
| Memory        | Net-Hirschmann-Standard-Memory-SNMP-custom    | Check device memory usage                           |

> | Processes     | Net-Hirschmann-Standard-Processes-SNMP-custom | Check process number of device                      |

Reviewer: ```suggestion
| Processes     | Net-Hirschmann-Standard-Processes-SNMP-custom | Check the number of processes for the device                      |
```

> The services listed above are created automatically when the **Net-Hirschmann-Standard-SNMP-custom** host template is used.

</TabItem>
<TabItem value="Not attached to a host template" label="Not attached to a host template">

| Service Alias        | Service Template                                         | Service Description                   |
|:---------------------|:---------------------------------------------------------|:--------------------------------------|
| Configuration        | Net-Hirschmann-Standard-Configuration-SNMP-custom        | Check configuration                   |

> | Traffic-Generic-Id   | Net-Hirschmann-Standard-Traffic-Generic-Id-SNMP-custom   | Check traffic of an network interface |

Reviewer: ```suggestion
| Traffic-Generic-Id   | Net-Hirschmann-Standard-Traffic-Generic-Id-SNMP-custom   | Check traffic of a network interface |
```

> | Traffic-Generic-Name | Net-Hirschmann-Standard-Traffic-Generic-Name-SNMP-custom | Check traffic of an network interface |

Reviewer: ```suggestion
| Traffic-Generic-Name | Net-Hirschmann-Standard-Traffic-Generic-Name-SNMP-custom | Check traffic of a network interface |
```

> | Traffic-Global       | Net-Hirschmann-Standard-Traffic-Global-SNMP-custom       | Check traffic of an network interface |

Reviewer: ```suggestion
| Traffic-Global       | Net-Hirschmann-Standard-Traffic-Global-SNMP-custom       | Check traffic of a network interface |
```

| hardware.temperature.count   | count |
| hardware.fan.count           | count |
| hardware.led.count           | count |
| hardware.psu.count           | count |

</TabItem>
<TabItem value="Memory" label="Memory">

| Name                    | Unit  |
|:------------------------|:------|
| memory.usage.bytes      | B     |
| memory.free.bytes       | B     |
| memory.usage.percentage | %     |

</TabItem>
<TabItem value="Processes" label="Processes">

| Name                            | Unit  |
|:--------------------------------|:------|
| processes.running.current.count | count |
| processes.running.maximum.count | count |

</TabItem>
<TabItem value="Traffic-*" label="Traffic-*">

| Name                                                 | Unit  |
|:-----------------------------------------------------|:------|
| status                                               | N/A   |
| *interface_name*#interface.traffic.in.bitspersecond  | b/s   |
| *interface_name*#interface.traffic.out.bitspersecond | b/s   |

> Applies to the following service templates: Traffic-Generic-Id, Traffic-Generic-Name, Traffic-Global

</TabItem>
</Tabs>

## Prerequisites

### SNMP Configuration

> To use this pack, the SNMP service must be properly configured on your **Hirschmann switch** equipment.

Reviewer: standard prerequisites

> To use this pack, the SNMP service must be properly configured on your **Hirschmann switch** equipment.

Author: ```suggestion
The SNMP agent must be configured and activated on the host. Please refer to the official documentation from the manufacturer/publisher.
```
